config/checkstyle/checkstyle-frames.xsl

```diff
@@ -57,8 +57,8 @@
    <http://www.apache.org/>.
```

Comment:
Are all these movement of lines necessary or is it because of auto formatting in intelliJ?

Reply:
All the config (checkstyle and pmd) files are copied from a new project. (NVA-NVI i think? @StigNorland knows). It makes sure the new version of the plugins still works with the configs

Reply:
Does this mean that all config xml from checkstyle, pmd etc are the same across all upgrade PR. If it is, i dont need to review every config file in every PR. It should be enough with reviewing one instance of those files?